configs/addons/counterstrikesharp/gamedata/gamedata.json

```diff
@@ -76,21 +76,21 @@
     "signatures": {
         "library": "server",
         "windows": "\\x48\\x8B\\xC4\\x4C\\x89\\x48\\x20\\x55\\x56",
-        "linux": "\\x55\\x48\\x89\\xE5\\x41\\x57\\x41\\x56\\x41\\x55\\x49\\x89\\xFD\\x41\\x54\\x53\\x48\\x81\\xEC\\xE8\\x01\\x00\\x00"
+        "linux": "\\x55\\x48\\x89\\xE5\\x41\\x57\\x41\\x56\\x41\\x55\\x49\\x89\\xFD\\x41\\x54\\x53\\x48\\x81\\xEC\\xE8"
       }
   },
   "UTIL_CreateEntityByName": {
     "signatures": {
         "library": "server",
         "windows": "\\x48\\x83\\xEC\\x48\\xC6\\x44\\x24\\x30\\x00\\x4C\\x8B\\xC1",
-        "linux": "\\x48\\x8D\\x05\\x49\\xB5\\xBC\\x00\\x55\\x48\\x89\\xFA\\x41\\x89\\xF0\\x48"
+        "linux": "\\x48\\x8D\\x05\\xC9\\xC2\\xBC"
```

Comment:
This one is quite short, confirmed working?

Reply:
yes it is working